server/banned.go

```diff
@@ -58,19 +63,25 @@ func (pi *packetInfo) checkBlameMessage() error {
 		// After validating everything, we can skip the actual ban
 		// if the pool already has banned someone.
 		if blamer.pool.firstBan != nil {
+			if debugMode {
+				fmt.Printf("[Blame] Ignoring blame in pool %d because a player is already banned\n", blamer.pool.num)
+			}
 			return nil
 		}
 
 		added := accused.addBlame(blamer.verificationKey)
 		if !added {
+			if debugMode {
+				fmt.Printf("[Blame] Duplicate blame A(%s) --> B(%s)\n", blamer, accused)
```

Comment:
No need for a long arrow. `fmt.Printf("[Blame] Duplicate blame from %s to %s\n", blamer, accused)`

Reply:
Ehehehe. About that. It's in parentheses because of the insane `String()` :D

I'll try to simplify both.